src/inverseRobot.js

```diff
@@ -24,7 +24,31 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  if (Object.keys(robot).length === 0) {
+    return {};
+  }
+
+  for (const firstEl in robot) {
+    let count = 0;
+
+    for (const secondEl in robot) {
+      if (robot[firstEl] === robot[secondEl]) {
+        count++;
+      }
+    }
+
+    if (count > 1) {
+      return null;
+    }
+  }
+
+  Object.entries(robot).forEach(([key, value]) => {
+    fixedRobot[value] = key;
+  });
```

Comment:
you can check if key is present for `fixedRobot` right here, go to [checklist](h to find out howttps://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)